geomesa-tools/bin/geomesa-kafka

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env bash
```

Comment:
good idea. When we do that we should consider adding the bin/geomesa-env.sh script similar to the tomcat's bin/set-env.sh and hadoops etc/hadoop/hadoop-env.sh etc.